api/find.go

```diff
@@ -0,0 +1,97 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/puppetlabs/wash/activity"
+	"github.com/puppetlabs/wash/api/rql"
+	"github.com/puppetlabs/wash/api/rql/ast"
+	apitypes "github.com/puppetlabs/wash/api/types"
+	"github.com/puppetlabs/wash/plugin"
+)
+
+// swagger:route GET /fs/find find listEntries
+//
+// Recursively descends the given path returning it and its children.
```

Comment:
This is where in-code API docs would go. But I think how to use this API should be its own section on our documentation site.

Reply:
Yeah, I'll work on that after this.